drivers/stepper/adi_tmc/Kconfig

```diff
@@ -26,30 +26,7 @@ config STEPPER_ADI_TMC_SPI
 
 comment "Trinamic Stepper Drivers"
 
-config STEPPER_ADI_TMC5041
-	bool "Activate trinamic tmc5041 stepper driver"
-	depends on DT_HAS_ADI_TMC5041_ENABLED && STEPPER_ADI_TMC
-	select STEPPER_ADI_TMC_SPI
-	default y
-	help
-	  Stepper driver for TMC5041.
-
-config STEPPER_ADI_TMC5041_RAMPSTAT_POLL
-	bool "TMC5041 poll ramp status"
-	depends on STEPPER_ADI_TMC5041
-	default y
-	help
-	  When enabled, the ramp status will be polled on TMC5041, to check for events:
-	  - TMC5041_POS_REACHED_EVENT
-	  - TMC5041_STOP_SG_EVENT
-	  - TMC5041_STOP_LEFT_EVENT
-	  - TMC5041_STOP_RIGHT_EVENT
-
-config STEPPER_ADI_TMC5041_RAMPSTAT_POLL_INTERVAL_IN_MSEC
-	int "TMC5041 poll ramp status interval in ms"
-	depends on STEPPER_ADI_TMC5041_RAMPSTAT_POLL
-	default 100
-	help
-	  The interval in ms to poll the ramp status on TMC5041.
+rsource "Kconfig.tmc22xx"
+rsource "Kconfig.tmc5041"
 
```

Comment:
Probably also added that keep-sorted comments?

Reply:
Done.